specs/v1.yaml

```diff
@@ -1595,7 +1914,7 @@ components:
               amount:
                 type: number
                 description: |-
-                  The final grant amount in cents that will be processed by Chariot and submitted to the DAF.
+                  The recurring grant amount in cents that will be processed by Chariot and submitted to the DAF.
```

Comment:
this seems like a mistake. please revert

Reply:
Thank you for catching.